de_web_plugin.cpp

```diff
@@ -498,6 +498,9 @@ struct lidlDevice {
 
 static const lidlDevice lidlDevices[] = { // Sorted by zigbeeManufacturerName
     { "_TYZB01_bngwdjsr", "TS1001",  "LIDL Livarno Lux", "HG06323" }, // Remote Control
+    { "_TZ1800_ejwkn2h2", "TY0203",  "LIDL Silvercrest", "HGXXXXX" }, // Contact sensor
+    { "_TZ1800_fcdjzz3s", "TY0202",  "LIDL Silvercrest", "HGYYYYY" }, // Motion sensor
```

Comment:
I guess the XXXX and YYYYY need to be filled?

The change from "SILVERCREST" to "LIDL Silvercrest" looks much better and I like that this is streamlined, but this already breaks one client ;) In the Phoscon App we match on manufacturer name for some devices, but we can add the change to support both.

We should notify other client devs (in case it's relevant for them).